From f007ba9e5ac0bbeee1c1d6b4e292b293629a838c Mon Sep 17 00:00:00 2001
From: Viktor Ashirov <vashirov@redhat.com>
Date: Wed, 17 May 2017 22:03:54 +0200
Subject: [PATCH] Issue 48864 - Fix FreeIPA build

Bug Description:
FreeIPA build fails because of incorrect include files

https://pagure.io/389-ds-base/issue/48864

Reviewed by: mreynolds (Thanks!)
---
 ldap/servers/slapd/slapi-plugin.h | 2 +-
 ldap/servers/slapd/slapi_pal.h    | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/ldap/servers/slapd/slapi-plugin.h b/ldap/servers/slapd/slapi-plugin.h
index ec8917d..4084945 100644
--- a/ldap/servers/slapd/slapi-plugin.h
+++ b/ldap/servers/slapd/slapi-plugin.h
@@ -28,7 +28,7 @@ extern "C" {
 #endif
 
 /* Provides our int types and platform specific requirements. */
-#include <slapi_pal.h>
+#include "slapi_pal.h"
 
 #include "prtypes.h"
 #include "ldap.h"
diff --git a/ldap/servers/slapd/slapi_pal.h b/ldap/servers/slapd/slapi_pal.h
index cb61d84..307679d 100644
--- a/ldap/servers/slapd/slapi_pal.h
+++ b/ldap/servers/slapd/slapi_pal.h
@@ -19,7 +19,9 @@
 
 #pragma once
 
+#ifdef HAVE_CONFIG_H
 #include <config.h>
+#endif
 
 #ifdef HAVE_INTTYPES_H
 #include <inttypes.h>
-- 
2.9.4

